<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;h1&gt;Calcular eliminação de Gauss</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calc">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #4CAF50; /* Green */
border-radius: 10px;
border: none;
color: white;
padding: 15px 32px;
text-align: center;
text-decoration: none;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="lbl_a11">
         <property name="styleSheet">
          <string notr="true">width: 35px;</string>
         </property>
         <property name="text">
          <string>&lt;h4&gt;0&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_a12">
         <property name="text">
          <string>&lt;h4&gt;0&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_a13">
         <property name="text">
          <string>&lt;h4&gt;0&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_a14">
         <property name="text">
          <string>&lt;h4&gt;0&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="lbl_a21">
         <property name="text">
          <string>&lt;h4&gt;0&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_a22">
         <property name="text">
          <string>&lt;h4&gt;0&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_a23">
         <property name="text">
          <string>&lt;h4&gt;0&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_a24">
         <property name="text">
          <string>&lt;h4&gt;0&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="lbl_a31">
         <property name="text">
          <string>&lt;h4&gt;0&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_a32">
         <property name="text">
          <string>&lt;h4&gt;0&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_a33">
         <property name="text">
          <string>&lt;h4&gt;0&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_a34">
         <property name="text">
          <string>&lt;h4&gt;0&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="lbl_x">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_y">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_z">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>535</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Elemento A11</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="matriz_a11">
           <property name="styleSheet">
            <string notr="true">width: 30px;</string>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Elemento A21</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="matriz_a21">
           <property name="styleSheet">
            <string notr="true">width: 30px;</string>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Elemento A31</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="matriz_a31">
           <property name="styleSheet">
            <string notr="true">width: 30px;</string>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Elemento A12</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="matriz_a12">
           <property name="styleSheet">
            <string notr="true">width: 30px;</string>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Elemento A22</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="matriz_a22">
           <property name="styleSheet">
            <string notr="true">width: 30px;</string>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Elemento A32</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="matriz_a32">
           <property name="styleSheet">
            <string notr="true">width: 30px;</string>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Elemento A13</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="matriz_a13">
           <property name="styleSheet">
            <string notr="true">width: 30px;</string>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Elemento A23</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="matriz_a23">
           <property name="styleSheet">
            <string notr="true">width: 30px;</string>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Elemento A33</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="matriz_a33">
           <property name="styleSheet">
            <string notr="true">width: 30px;</string>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Elemento A14</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="matriz_a14">
           <property name="styleSheet">
            <string notr="true">width: 30px;</string>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Elemento A24</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="matriz_a24">
           <property name="styleSheet">
            <string notr="true">width: 30px;</string>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Elemento A34</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="matriz_a34">
           <property name="styleSheet">
            <string notr="true">width: 30px;</string>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;h2&gt;A =&lt;/h2&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
